chain/vm/vm.go

```diff
@@ -210,15 +210,12 @@ func (vm *VM) send(ctx context.Context, msg *types.Message, parent *Runtime,
 		}()
 	}
 
-	aerr := rt.chargeGasSafe(rt.Pricelist().OnMethodInvocation(msg.Value, msg.Method))
```

Comment:
This should be `chargeGasSafe` otherwise `ApplyMessage` can panic as it has no panic handler installed.

Reply:
Ah, right, thanks!